### ISSUE TITLE
Plot loss Mask R-CNN

### ISSUE THREAD

**Issue by Ujang24** [matterport/Mask_RCNN#1072]
As in any other DL, we need to train with **a lot of epoch** to have a good result.
One way to evaluate our model is using loss values. 

Anyone knows where does loss values are stored?
There are some loss values given by Mask R-CNN. How to plot it?

**Comment by keineahnung2345** [comment]
You can use tensorboard. Just type `tensorboard --logdir=<your-log-dir>` in command line, and go to <http://your-ip:6006> using a browser.

**Comment by Ujang24** [comment]
Thank you for the answer. I have tried that command line. But I got this message when I opened the browser:
```
This site can’t be reached desktop-fhr24ns refused to connect.
Search Google for desktop fhr24ns 6006
ERR_CONNECTION_REFUSED
```

I tried once again the command line, now it even cant do the process with this error
```
(tensorflow) D:\S3\Sandwich-Program\Program\Mask-RCNN-Matterport\samples\balloon>tensorboard --logdir=logs
Unable to create process using 'C:\Users\Lenovo\Anaconda3\envs\tensorflow\python.exe C:\Users\Lenovo\Anaconda3\envs\tensorflow\Scripts\tensorboard-script.py --logdir=logs'
```
PS: in the documentation, it said that I don't have to give "=". I have tried with and without "=" but both gave errors.

Any help will be appreciated

**Comment by keineahnung2345** [comment]
Maybe http://127.0.0.1:6006 will work, take it a try!

**Comment by Ujang24** [comment]
Thank you for the suggestion. 
It gave the same error.

**Comment by keineahnung2345** [comment]
Check this out: https://github.com/tensorflow/tensorflow/issues/9701

**Comment by Samb55** [comment]
I have same issue with tensoboard. I used `tensorboard --logdir=Mask_RCNN/logs/cats20190221T1603` and generate the link. However, when I copy the link and paste it in my browser, it show me the browser can't connect to the server.

**Comment by jayhou07** [comment]
I met the same problem using win10 64bit. And as @keineahnung2345 said, that issue contained the solution: use `tensorboard --logdir=<your-log-dir> --host=localhost`.

**Comment by loyanie** [comment]
Hello, I'm using matterport mask rcnn for instance segmentation but I would like to know how to plot loss and mAP graphs against epochs respectively
